mayastor-test/e2e/example-simple.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
```

Comment:
just a nit. using env with bash would be more portable (applies to all bash scripts in this PR).